lib/improver/uv_index.py

```diff
@@ -34,19 +34,19 @@
 from cf_units import Unit
 
 
-def calculate_uv_index(uv_upward, uv_downward, scale_factor):
+def calculate_uv_index(uv_upward, uv_downward, scale_factor=3.6):
```

Comment:
It was decide previously that the scale factor would not be set here, but would be set in the CLI.